docs/reference/data-streams/logs.asciidoc

```diff
@@ -1,18 +1,18 @@
 [[logs-data-stream]]
 == Logs data stream
 
-preview::[Logs data streams and the logsdb index mode are in tech preview and may be changed or removed in the future. Don't use logs data streams or logsdb index mode in production.]
+IMPORTANT: This feature requires an https://www.elastic.co/pricing[Enterprise subscription].
```

Comment:
Maybe reword this? Logsdb doesn't require an Enterprise license. Without an Enterprise license stored source will be used instead of synthetic source. So an enterprise license in the case of logsdb should result into less storage usage.

Reply:
I think we generally delegate to the [subscriptions page](https://www.elastic.co/subscriptions) for this info rather than being explicit in the docs. I'm guessing that page will be updated by PMM (or whoever owns that) for 8.17?

Reply:
Perhaps combine this note with availability information:

```asciidoc
The Elasticsearch `logsdb` index mode is generally available for Elastic Cloud Hosted 
and Self-Managed customers starting in version 8.17 and is enabled by default for 
logs in https://www.elastic.co/elasticsearch/serverless[Elastic Cloud Serverless].
```